### ISSUE TITLE
Other A11y: Many Pages: Various fields have no visible name to explain their purpose

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76927]
## Action Performed:
Prerequisites:
- The user is signed in.

1. Using Chrome, open the URL - https://new.expensify.com/
2. Navigate to the "+" icon and press Enter
3. Tab to the "Track distance" menu item and press Enter to open the Pop-up
4. Tab to the "Manual" Tab and press Enter to select
5. Observe the input field visually, and there is no visible label provided for the input field

Other Occurrences:
1. On Send Invoice
2. On Create Expense

## Expected Result:
Every form field must have a visible text label that clearly explains its purpose, so that the users can understand the purpose and what it describes in the input field, e.g.:
- 'distance' or similar

## Actual Result:
The input field has no visible name/label explaining its purpose of the input field; users only see an unlabeled field or placeholder text, as users do not know what information is required.

HTML:
<div dir="auto" class="css-146c3p1" style="color: rgb(175, 187, 176); font-size: 40px; text-align: left; font-family: &quot;Expensify New Kansas&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 500;">mi</div>

## Area issue was found in:
Track Distance

## Failed WCAG checkpoints
WCAG 3.3.2 Labels or Instructions
https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions

## User impact:
Providing clear and unambiguous labels and instructions (including examples of expected data formats) helps all users - but particularly those with cognitive, language, and learning disabilities - to enter information correctly and can prevent users from making incomplete or incorrect form submissions.

## Suggested resolution:
Ensure the field has a visible name to clearly describe its purpose, e.g.:
- 'Distance'
https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions

## Environment(s)
Windows Windows 11 Home - Chrome

## Screenshots/Videos
<img width="1919" height="969" alt="Image" src="https://github.com/user-attachments/assets/5d88435c-7ba2-4dff-b825-fd17075bc61f" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6997043
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
> For the distance one, I do agree that we need to read out the correct label, but can't this be done with a aria-label? We don't need a visible label for it given visually it's on the top of the page, but for screen readers I agree that it's too attached. I'd just put the label as a screen reader only label right before the input?

alright, we've decided that we are gonna keep this accessibilitylabel only and not add form label to the amount input


source: https://expensify.slack.com/archives/C01GTK53T8Q/p1773351708274109?thread_ts=1773346127.842749&cid=C01GTK53T8Q

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544083 (Web)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/d0bfff593092b9a6e958388c30249f0ff920c2a875e2681209b40f0734fe7222
- https://platform.applause.com/services/links/v1/external/28551e7189e9e69eed092e4c1489a848ac6ddb7996da2adb711266d04a29f6de
- https://platform.applause.com/services/links/v1/external/699e5743fbfda68bc9be86670cb5320af0eb9db9cac20db7540e36587238bbbb

### Retest notes
Not fixed: The input field has no visible name/label explaining its purpose of the input field; users only see an unlabeled field or placeholder text, as users do not know what information is required.

Not fixed :
Other Occurrences:
1. On Send Invoice
2. On Create Expense

**Comment by rushatgabhane** [comment]
Closing as fixed https://expensify.slack.com/archives/C029ZDBREUV/p1778609001235259
